docs/v1/use-cases.md

```diff
@@ -17,37 +17,121 @@ implement.
         be able to publish my services using a service broker, so that users
         external to my Kubernetes cluster can use my service
 
-### Blackbox services
+### Sharing blackbox services
 
 There are numerous SaaS providers that already operate service brokers today.
 It should be possible for the operator of an existing service broker to
 publish their services into the catalog and have them consumed by users of
 Kubernetes.  This offers a new set of users to the service operator and offers
 a wide variety of SaaS products to users of Kubernetes.
 
-### Exposing existing services outside the cluster
+### Exposing Kubernetes services outside the cluster
 
 It should be possible for service operators whose services are deployed in a
 Kubernetes cluster to publish their services using a service broker.  This
 would allow these operators to participate in the existing service broker
 ecosystem and grow their user base accordingly.
 
-## Low-Level Use Cases
+## CF Service Broker `v2` API Use Cases
 
-These are lower-level use cases the service catalog will implement in service
-to the high-level use cases.
+Initially, the catalog should support the current [CF Service Broker
+API](https://docs.cloudfoundry.org/services/api.html) These are the use cases
+that the service catalog has to implement in order to use that API.
 
-1.  As a service broker operator, I want to be able to register my broker with
+### Managing service brokers
+
+1.  As a catalog operator, I want a user to be able to register a broker with 
     the Kubernetes service catalog, so that the catalog is aware of the services
-    my broker offers
+    that broker offers
+2.  As a catalog operator, I want a user to be able to update a registered
+    broker so that the catalog can maintain the most recent versions of services
+    that broker offers
+3.  As a catalog operator, I want to be able to delete a broker from the 
+	catalog, so that I can keep the catalog clean of brokers I no longer want to
+	use
```

Comment:
The indenting needs to align with the lines above.
And, s/to use/to support/ might be more appropriate.  But no biggie either way.